core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -679,6 +681,34 @@ export class WorkflowEditorComponent implements AfterViewInit, OnDestroy {
       });
   }
 
+  private handleCommentBoxHighlightMouseClickInput(): void {
+    // on user mouse clicks a comment box, highlight that comment box
+    fromJointPaperEvent(this.getJointPaper(), "cell:pointerdown")
+      //fromJointPaperEvent(this.getJointPaper(), "cell:contextmenu")
```

Comment:
Either delete this comment, or write it as in `handleHighlightMouseInput`. Actually this piece of code should not be wrote separately. Consider merging this into `handleHighlightMouseInput` to prevent duplicate code.